src/easylink/steps/dev/python_pandas/dummy_step.py

```diff
@@ -22,33 +22,25 @@ def load_file(file_path, file_format=None):
     raise ValueError()
 
 
-diagnostics = {}
-
-if "DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS" in os.environ:
-    main_input_file_paths = os.environ["DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS"].split(",")
-else:
-    main_input_file_paths = glob.glob("/input_data/main_input*")
-
-logging.info("Loading main input")
-diagnostics["num_main_input_files"] = len(main_input_file_paths)
-df = load_file(main_input_file_paths[0])
+INPUT_ENV_VARS = os.getenv(
+    "INPUT_ENV_VARS",
+    "DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS",
```

Comment:
Will this default (not including secondary) change the behavior of some implementations?

Reply:
I think as long as you put e.g.
```
  env:
    INPUT_ENV_VARS: "DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS,DUMMY_CONTAINER_SECONDARY_INPUT_FILE_PATHS"
```
in the implementation metadata for the relevant steps the result should be the same (and should be confirmed by the checksum in the e2e tests)